src/js/client/coverageSender.js

```diff
@@ -15,85 +15,55 @@
 
  */
 /* globals Testem */
-(function (fluid, $, Testem) {
+(function (Testem) {
     "use strict";
+    // Pure JS equivalent of several fluid.registerNamespace calls.
+    window.gpii = { testem: { coverage: {} } };
```

Comment:
I discovered a deep flaw with this today that would have bitten us soon enough.  It's only truly safe if you're working outside the gpii namespace or loading the coverage client absolutely last.  Otherwise it's very easy for it to delete existing namespaced material.  I'll clean this up before I cut the release.